quinn-proto/src/endpoint.rs

```diff
@@ -154,6 +154,7 @@ where
         &mut self,
         now: Instant,
         remote: SocketAddr,
+        initial_local_ip: Option<IpAddr>,
```

Comment:
```suggestion
        local_ip: Option<IpAddr>,
```
At this layer we have no notion of initial vs. not regardless.